### DISCUSSION TITLE
Zoraxy not update

### DISCUSSION THREAD

**Discussion by barcelona92** [tteck/Proxmox#3083] [General]
Hi,

The Zoraxy in the Github appear a new version, but how too update LXC?

I update with script LXC updater bit is not update.

Thx

**Comment by tteck** [comment]
Try typing the command `update` within the LXC console.

**Comment by barcelona92** [comment]
Thx @tteck but not update , appear the same version...

**Reply by tteck** [reply]
Please try again, https://github.com/tteck/Proxmox/commit/a95657947db0f0c183422a95b09a2f4a919cacca

**Reply by cvroque** [reply]
I managed to update normally on my installation, before your latest commit. 

If the issue persists, maybe a git stash will be necessary like it's done in the zigbee2mqtt update script.

**Reply by tteck** [reply]
easy enough to test @barcelona92 

```
systemctl stop zoraxy
cd /opt/zoraxy/src
git stash
git pull
go mod tidy
go build
systemctl start zoraxy
```

**Reply by barcelona92** [reply]
error:

hint: Pulling without specifying how to reconcile divergent branches is
hint: discouraged. You can squelch this message by running one of the following
hint: commands sometime before your next pull:
hint: 
hint:   git config pull.rebase false  # merge (the default strategy)
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint: 
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
fatal: unable to access 'https://github.com/tobychui/zoraxy/': Could not resolve host: github.com

**Comment by barcelona92** [comment]
Ohh i all ok, reinstalling the LXC i updated 2 version, but the last update appear this error:


 |hint: Pulling without specifying how to reconcile divergent branches is
hint: discouraged. You can squelch this message by running one of the following
hint: commands sometime before your next pull:
hint:
hint:   git config pull.rebase false  # merge (the default strategy)
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint:
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
 /error: Your local changes to the following files would be overwritten by merge                                                                             :
        src/go.mod
        src/go.sum
Please commit your changes or stash them before you merge.
Aborting
 ✓ Updated Zoraxy

**Comment by KmdKeen** [comment]
Update to 3.0.8 is not working because golang in v1.22 is needed. With removing 1.19 and installing 1.22 manunally the update is working for me.
